deps/rabbitmq_auth_backend_oauth2/test/jwks_SUITE.erl

```diff
@@ -0,0 +1,390 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Not really sure that it's great to copy/paste all of `system_SUITE`, but it was one way to avoid interfering with the common broker (which could break other tests, or at the very least change their meaning significantly).

Reply:
Having a separate suite is perfectly fine and what I would do 👍 . Test suites generally start nodes that are not shared by any other suites. The same can be done for individual groups of tests or even individual tests. See the [main quorum queue suite](https://github.com/rabbitmq/rabbitmq-server/blob/master/deps/rabbit/test/quorum_queue_SUITE.erl), it has quite a few extensive examples.

This way you do not affect other suites (groups, tests).

Reply:
Alright, closing this thread then.